modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/search/SearchSearchRequestAssemblerImpl.java

```diff
@@ -192,6 +193,16 @@ private void _setPreference(
 		}
 	}
 
+	private void _setSearchAfter(
+		SearchSourceBuilder searchSourceBuilder,
+		SearchSearchRequest searchSearchRequest) {
+
+		if (searchSearchRequest.getSearchAfter() != null) {
```

Comment:
it is a Object[] so it must check `.lenght != 0`

Reply:
:+1: